Upstream: Wants it the way it is for their development.
Source: philantrop@exherbo.org
Summary: The test "regress" needs a local web server and can be safely disabled.

diff -Naur libevent-1.4.3-stable.orig/test/test.sh libevent-1.4.3-stable/test/test.sh
--- libevent-1.4.3-stable.orig/test/test.sh	2007-11-12 03:37:31.000000000 +0100
+++ libevent-1.4.3-stable/test/test.sh	2008-05-08 04:58:32.000000000 +0200
@@ -40,12 +40,13 @@
 	echo FAILED ; 
 fi
 echo -n " regress: "
-if ./regress >/dev/null ; 
-then 
-	echo OKAY ; 
-else 
-	echo FAILED ; 
-fi
+#if ./regress >/dev/null ; 
+#then 
+#	echo OKAY ; 
+#else 
+#	echo FAILED ; 
+#fi
+echo SKIPPED
 }
 
 echo "Running tests:"
